oak_runtime/src/introspection_browser_client/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# // TODO(#913): Implement introspection client
+
+dir=./dist
+mkdir -p $dir
+
+indexjs=$dir/index.js
+indexhtml=$dir/index.html
+
+test -f $indexjs || touch $indexjs
```

Comment:
```suggestion
[[ -f $indexjs ]] || touch $indexjs
```

https://google.github.io/styleguide/shellguide.html#test----and---

also maybe make it look more like an actual `if` (not sure how to do negation though, is `!` enough?)

Reply:
Yup, `!` is enough :)